### PR TITLE
Popup title and close button no longer participate in scrolling.

### PR DESCRIPTION
**Pull request by anmacdonald**
Additional tweaks based on your feedback. Popup title and close button no longer participate in the scroll behaviour.

https://user-images.githubusercontent.com/42203545/143486512-9fa48d53-adf2-4b6f-b592-56d436564b03.mp4

### CODE REVIEW

File: uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml [hunk 1]
```diff
@@ -113,7 +112,24 @@ Pane {
...
+    spacing: 5
+    leftPadding: popupView.spacing
+    rightPadding: popupView.spacing
+
+    title: controller.title
+
+    // Title Header
+    header: Label {
+        textFormat: Text.StyledText
+        text: `<h2>${popupView.title}</h2>`
```
Comment (mrwillyees) [inline uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml:124]: ❓ why using `popupView.title` and not directly `controller.title`?

File: uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml [hunk 2]
```diff
@@ -190,7 +193,10 @@ Pane {
                 Label {
                     Layout.fillWidth: true
                     text: formattedValue
-                    wrapMode: Text.WrapAnywhere
+                    wrapMode: Text.Wrap
+                    palette: popupView.palette
+                    font: popupView.font
+                    horizontalAlignment: Text.AlignRight
```
Comment (mrwillyees) [inline uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml:199]: 💭 I think I preferred with a left align, it is a bit easier to read and it is more table-like

**Review by mrwillyees — APPROVED** [review]
LGTM, so it was a matter of setting the header and footer outside from the scrollable content! It looks much much better with your tweaks

File: uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml (hunk 1 above)
Comment (anmacdonald) [reply uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml:124]: The full code is this:

```qml
    title: controller.title

    // Title Header
    header: Label {
        textFormat: Text.StyledText
        text: `<h2>${popupView.title}</h2>`
```

`Page` comes with a title property, so it makes sense to populate it with the data so if read it shows something sensible, it also allows the user to override it and see the effect take place.

File: uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml (hunk 2 above)
Comment (anmacdonald) [reply uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml]: Very fair, I have reverted back!

**Pull request merged**